src/univers/version_range.py

```diff
@@ -774,7 +776,8 @@ class GolangVersionRange(VersionRange):
         ">=": ">=",
         "<": "<",
         ">": ">",
-        "=": "=",  # This is not a native golang-semver comparator, but is used in the gitlab version range for go packages.
+        "=": "=",
+        # This is not a native golang-semver comparator, but is used in the gitlab version range for go packages.
```

Comment:
Is these such thing as golang semver?
Also I wonder if we should not get rather a gitlab-specific preprocessor rather than updating all range implementations?